docs/developer-interface.rst

```diff
@@ -25,6 +25,12 @@ Dataset Status
 .. autoclass:: tamr_unify_client.models.dataset_status.DatasetStatus
   :members:
 
+Dataset Profile
```

Comment:
could you move this to be before `DatasetStatus` so we maintain alphabetic ordering?